phpunit.xml

```diff
@@ -9,7 +9,7 @@
             <directory suffix="Test.php">./tests/Tenant</directory>
             <directory suffix="Test.php">./app-modules/*/tests/Tenant</directory>
         </testsuite>
-    </testsuites>
+    <testsuite name="Modules"><directory suffix="Test.php">./app-modules/*/tests</directory></testsuite></testsuites>
```

Comment:
What is the purpose of this change?

The module tests are already called properly above and all tests should be under the Tenant or Landlord folders.

Are there any that exist now that aren't, if so let me know. If any you added are not, please move them. Most likely based on this PR, under Tenant.